configuration/configure-inactive-cluster-removal.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="configure-inactive-cluster-removal"]
+= Configuring Removal of Inactive Clusters
+include::modules/common-attributes.adoc[]
+:context: configure-inactive-cluster-removal
+
+toc::[]
+
+[role="_abstract"]
+{product-title} provides the ability to automatically remove, or decommission, inactive clusters in your system. Deleting inactive clusters allows you to monitor only active clusters. You can configure cluster decommissioning settings by using the {product-title-short} portal or the application programming interface (API).
+
+This setting is disabled by default. To enable this setting, configure a value in the *Cluster Decommissioning* field, as described in the following procedure. If the setting to remove inactive clusters is enabled, when a cluster is unreachable, the system displays a warning that the cluster is scheduled for deletion in seven days. After a cluster is removed, the removal is documented with an `info` log in the Central logs.
```

Comment:
> This setting is disabled by default

This is not true is it Charmik?

Reply:
Yeah, I totally guessed based on the JIRA so please confirm :) (this will go through official review later so we will get to it then if not now)